view/frontend/templates/breadcheckout/configurable.phtml

```diff
@@ -29,13 +30,9 @@
                     validSuperAttribute = validSuperAttribute + Boolean(selectedOptions[attributeId]);
                 });
                 validSuperAttribute =  Boolean(validSuperAttribute && validSuperAttribute.search("false") === -1);
-                /**
-                 * Validate add to cart form
-                 */
-                $.mage.validation({errorPlacement:function () {},highlight:function () {}}, form);
```

Comment:
not sure exactly what this mage validation is, but just to clarify, with this change we'll not be using it at all anymore, and doing it ourselves, right?

Reply:
Yes, that's not being used anymore from this change onwards